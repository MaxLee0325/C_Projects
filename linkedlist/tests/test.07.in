new
append alpha
append beta
append gamma
iterator
next
next
remove
next
remove
end
