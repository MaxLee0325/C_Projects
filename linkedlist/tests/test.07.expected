[ ]
[ alpha ]
[ alpha beta ]
[ alpha beta gamma ]
[ alpha beta gamma ]
Next: alpha
[ alpha beta gamma ]
Next: beta
[ alpha beta gamma ]
Removed: 'beta'
[ alpha gamma ]
Next: gamma
[ alpha gamma ]
Removed: 'gamma'
[ alpha ]
